client/src/pages/Home.js

```diff
@@ -1,44 +1,52 @@
-import { Link } from "react-router-dom";
+// import { Link } from "react-router-dom";
 import Search  from "../components/Search.jsx";
+import JobCardContainer from "../components/JobCardContainer.jsx";
 import React, { useEffect, useState } from "react";
 import axios from "axios";
 
 import "./Home.css";
 
 function Home() {
-  const [jobs, setJobs] = useState([]);
+	const [jobs, setJobs] = useState([]);
```

Comment:
Great job, @SanaAsaf!

It seems that there is a conflict with the current main branch because you made changes to the Home() function.

To resolve this, I suggest bringing back the latest version of the Home() function from the main branch. Once that is done, you can add the job container somewhere on the page.

Since we currently do not have any data to show inside the container, you can leave it blank for now. We'll pass the necessary parameters later on when we have the data ready. This way you also submit more PRs! 🚀